docs/source/api_reference/models.rst

```diff
@@ -0,0 +1,206 @@
+Models
+======
+
+All resources in the API should be represented by a model class exported by
```

Comment:
See if you can move this to ``models.__init__``. We can put that into an ``automodule`` directive here.